### ISSUE TITLE
Trying to load the Android app on a school manged chrombook

### ISSUE THREAD

**Issue by Faltermeier** [OpenOrienteering/mapper#1229]
I get a message to open OS Developer mode as this android app is not from the Play store 

### Expected behaviour
How can this be resolved as we use school managed Chromebooks that can run Android apps 

### Configuration
Mapper Version:0.8.4
Operating System: Android

**Comment by dg0yt** [comment]
I don't think we are ready for the Play store at the moment, and running Mapper on Chromebooks is untested.

So the only installation method on Android and on Chromebooks is 'sideloading' which [requires the developer mode on Chromebooks](https://developer.android.com/topic/arc/sideload), indeed. (Note that at work or school, even [the Play store might not be available](https://support.google.com/chromebook/answer/7021273?hl=en).)
